ruby/basic_ruby/predicate_enumerable_methods.md

```diff
@@ -39,12 +40,14 @@ result = false
 numbers.each do |number|
   if number == element
     result = true
+    break
   end
 end
 
 result
 #=> false
 ```
+If you notice, there is a `break` statement in the `if` condition so that loop stops executing once the number is found. Otherwise it would still run over all numbers in the array which is not needed as we have already found the element
```

Comment:
Nit: This would be a good spot for one of our note boxes:

```suggestion
<div class="lesson-note">
If you notice, there is a `break` statement in the `if` condition so that loop stops executing once the number is found. Otherwise it would still run over all numbers in the array which is not needed as we have already found the element.
</div>
```

Reply:
done